langDir/en.json

```diff
@@ -751,5 +751,13 @@
     "understand": "I understand",
     "learnText": "Learn how to Connect",
     "downloadExtension": "Download extension"
+  },
+  "actionLabels": {
+     "MAKE_OFFER": "MAKE OFFER",
+     "BUY": "BUY",
+     "SEND": "SEND",
+     "CONSUME": "CONSUME",
+     "DOWNLOAD": "DOWNLOAD",
+     "LIST": "LIST"
   }
```

Comment:
don't we already have this in `nft.envent`?

Reply:
I did reuse it but I am not sure if we can, because we use this here to map it, and maybe if we remove the underscore, it wont map correctly, but i am not sure if thats the case can someone confirm. 
<img width="695" alt="Screen Shot 2022-06-29 at 6 21 33 PM" src="https://user-images.githubusercontent.com/39299315/176572556-a487185a-a33e-4bd2-8176-8eb232616a62.png">

Reply:
cool then we can close this